.circleci/config.yml

```diff
@@ -1,11 +1,14 @@
 version: 2.1
 
 orbs:
+  architect: giantswarm/architect@dev:lint-changelog
```

Comment:
I take it this will need to be changed after merging?

Reply:
Yes.

Reply:
:+1: